pkgs/development/libraries/science/math/tensorrt/extension.nix

```diff
@@ -38,13 +38,13 @@ final: prev: let
     + ".tar.gz";
 
   tensorRTVersions = {
-    "8.6.0" = [
+    "8.6.1" = [
```

Comment:
We might want to upgrade from `tensorrtVersions : TRTVersion -> Source` to `TRTVersion -> CUDAVersion -> Source` to `[(TRTVersion, CUDAVersion, Source)]`, and we need to document a way for users to override the scope with their own trt version, out of tree. OTOH, https://github.com/NixOS/nixpkgs/pull/256324 right now is experimenting on solving a similar problem (using `evalModules`) so maybe we don't refactor until we see how that plays out

Reply:
I agree that the current architecture lacks flexibility.
Maybe we can keep going with this change and leave the refactor for another PR. What do you think ?

Reply:
I think so too